doc/_examples/pong.cc

```diff
@@ -9,28 +9,28 @@
 using namespace broker;
 
 int main() {
-    // Setup endpoint and connect to Zeek.
-    endpoint ep;
-    auto sub = ep.make_subscriber({"/topic/test"});
-    auto ss = ep.make_status_subscriber(true);
-    ep.listen("", 9999);
+  // Setup endpoint and connect to Zeek.
+  endpoint ep;
+  auto sub = ep.make_subscriber({"/topic/test"});
+  auto ss = ep.make_status_subscriber(true);
+  ep.listen("", 9999);
 
-    // Wait until connection is established.
-    for ( bool has_peer = false; !has_peer; ) {
```

Comment:
This was a weird mix of the Zeek style and whatever broker uses by default. There's actually a flag you can pass to clang-format to preserve this style, if that's what we want to do (`SpacesInConditionalStatement`). It doesn't look like we prefer this format everywhere else though, so I'm fine with letting it go.